### PR TITLE
Tweak XCM slides for PBA4

### PR DESCRIPTION
**Pull request by franciscoaguirre**
- Move some content from lesson 1 to lesson 2 since it was too big
- Improve XCVM execution diagrams (lecture 2)
- Simplify pallet-xcm slides (lecture 3)
- Totally redesigned lecture 4 to focus on an end-to-end scenario

### CODE REVIEW

File: syllabus/8-XCM/2-XCVM/XCVM_slides.md [hunk 1]
```diff
@@ -143,37 +328,175 @@ Notes:
...
+
+Notes:
+
+The two ways of transferring assets between consensus systems are teleports and reserve transfers.
+
+---v
+
+### 1. Asset teleportation
+
+<img rounded style="width: 500px;" src="../../../assets/img/8-XCM/teleport.png" />
```
Comment (wirednkod) [inline syllabus/8-XCM/2-XCVM/XCVM_slides.md:419]: Please move the images inside the 8-XCM directory, and pick them from there (delete the old ones).
After that move the above should be:
```suggestion
<img rounded style="width: 500px;" src="../img/8-XCM/teleport.png" />
```

File: syllabus/8-XCM/2-XCVM/XCVM_slides.md [hunk 2]
```diff
@@ -143,37 +328,175 @@ Notes:
...
+<diagram class="mermaid">
+graph LR
+    Polkadot(Polkadot)--"No trust"-->Kusama(Kusama)
+</diagram>
+
+---v
+
+### 2. Reserve asset transfers
+
+<img rounded style="width: 400px;" src="../../../assets/img/8-XCM/reserve-tx.png" />
```
Comment (wirednkod) [inline syllabus/8-XCM/2-XCVM/XCVM_slides.md:448]: Please move the images inside the 8-XCM directory, and pick them from there (delete the old ones).
After that move the above should be:
```suggestion
<img rounded style="width: 500px;" src="../img/8-XCM/reserve-tx.png" />
```

**Review by wirednkod — CHANGES_REQUESTED** [review]

**Review by wirednkod — DISMISSED** [review]
link-wise, code-wise and assets-wise that looks good.
@KiChjang will have to check the content I guess

File: syllabus/8-XCM/2-XCVM/XCVM_slides.md [hunk 3]
```diff
@@ -42,90 +83,235 @@ XCVM Instructions might change a register, they might change the state of the co
 
 ---v
 
-## Example: TransferAsset
+## Example: WithdrawAsset
 
-An instruction used to transfer assets to some other address.
+An instruction used to get assets from an account into the holding register.
```
Comment (KiChjang) [inline syllabus/8-XCM/2-XCVM/XCVM_slides.md:88]:
```suggestion
An instruction used to get assets from an account and put it into the holding register.
```

File: syllabus/8-XCM/2-XCVM/XCVM_slides.md [hunk 4]
```diff
@@ -42,90 +83,235 @@ XCVM Instructions might change a register, they might change the state of the co
...
+        BuyExecution-->DepositAsset
     end
 </diagram>
 
 Notes:
 
-Registers _are_ the state of XCVM.
-Note that they are temporary/transient.
-We'll talk about are the `holding` and `origin` registers, but there are more.
+Here we have a very simple program, which if executed locally would result in a regular asset transfer.
```
Comment (KiChjang) [inline syllabus/8-XCM/2-XCVM/XCVM_slides.md:158]:
```suggestion
Here we have a very simple program, which if executed locally would result in a regular asset transfer within a single consensus system.
```

File: syllabus/8-XCM/2-XCVM/XCVM_slides.md [hunk 5]
```diff
@@ -143,39 +329,177 @@ Notes:
...
+    AssetHub-->Alice(Alice)
+    AssetHub-->AssetsPallet(Pallet Assets)
+    AssetsPallet-->Asset(USDT)
+</diagram>
+
+Notes:
+
+Locations are relative, so they must be updated and rewritten when sent to another chain, for them to be interpreted correctly.
+
+TODO: Move elsewhere.
```
Comment (KiChjang) [inline syllabus/8-XCM/2-XCVM/XCVM_slides.md:351]: Move elsewhere?

**Review by wirednkod — APPROVED** [review]

**Review by KiChjang — APPROVED** [review]

**Pull request merged**